src/d3a/models/market/basic_market.py

```diff
@@ -0,0 +1,71 @@
+"""
+Copyright 2018 Grid Singularity
+This file is part of D3A.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+from logging import getLogger
+
+from d3a.models.market.two_sided_pay_as_bid import TwoSidedPayAsBid
+from d3a.models.market.market_structures import TradeBidOfferInfo
+from d3a_interface.constants_limits import ConstSettings
+
+log = getLogger(__name__)
+
+DEFAULT_PRECISION = 8
+FLOATING_POINT_TOLERANCE = 0.00001
+
+
+class BasicMarket(TwoSidedPayAsBid):
```

Comment:
Instead of having a separate market for that, why not making Mycos the default behavior for all markets ? That way we can deprecate 2 sided pay as clear market, and have only 2 market types (one sided, two sided) and 2 myco types (pay as bid , pay as clear)

Reply:
Adapted, please give it a look: https://github.com/gridsingularity/d3a/pull/1121/commits/30d621c9303d6bcb5c5ef9e9a8516a8e787fd440